src/Versity.Users/External/Infrastructure/InfrastructureInjection.cs

```diff
@@ -40,16 +42,18 @@ public static async Task<IServiceProvider> EnsureRolesExists(this IServiceProvid
         var anyRoleWasAdded = false;
         foreach (var role in roles)
```

Comment:
Not the best way to add roles

Reply:
Actually, i think i even dont need to manualy add roles here, bc ASP.Core Identity already comes with roles, such as "Administrator" or "Member".

Reply:
And I do not know how to do it differently :\

Reply:
You can create something like data seeder and here add neccessary for your app roles and etc

Reply:
I made a data seeder for [roles](https://github.com/jejikeh/versity/pull/17/files#diff-0b0e7c1a0378497d9a445905ecd9d18a4b9a7b4d4b53ad992af976a712df5130), [admin](https://github.com/jejikeh/versity/pull/17/files#diff-364cc66a27706bc78600c4f821c48f3e0a60a30a3fd9bc60d72e2eef9e169646) and [userRoles](https://github.com/jejikeh/versity/pull/17/files#diff-20cdd38d5bffa7b157beaaaaf089d0213c1c6e4ca034f0008f987ae2bd4e65c6)